build.gradle

```diff
@@ -207,6 +207,9 @@ configure(projectsWithFlags('java')) {
         // Jetty ALPN support
         compileOnly libs.jetty.alpn.api
 
+        // Snappy
+        implementation libs.snappy
```

Comment:
Oops, I realized that this should be in core/build.gradle

Reply:
Had a chat with @minwoox and I realized Netty already supports Snappy encoder out of the box. The snappy dependency has been removed from the runtime classpath.